admin/onpage/class-onpage.php

```diff
@@ -205,4 +210,21 @@ private function catch_redo_listener() {
 		}
 	}
 
+	/**
+	 * Checks if WordFence protects the site against 'fake' Google crawlers.
+	 *
+	 * @return boolean
+	 */
+	private function wordfence_protection_enabled() {
+		if ( ! class_exists( 'wfConfig' ) ) {
+			return false;
+		}
+
+		if ( ! method_exists( 'wfConfig', 'get' ) ) {
+			return false;
+		}
+
+		return ! ! wfConfig::get( 'blockFakeBots' );
```

Comment:
Could just be me, but `! !` ? Does that make sure it's cast to `bool` or something?